roles/pulpcore_repositories/tasks/main.yml

```diff
@@ -24,4 +24,5 @@
     creates: /etc/dnf/modules.d/pulpcore.module
   when:
     - pulpcore_repositories_version is version('3.16', '>=')
+    - pulpcore_repositories_version is version('3.39', '<')
```

Comment:
I think this is correct.

On a related note: I wonder if we need to disable the module on `>= 3.39`.